src/soundsourceproxy.cpp

```diff
@@ -219,24 +170,18 @@ void SoundSourceProxy::loadPlugins() {
     s_soundSourceProviders.registerProvider(Mixxx::SoundSourceProviderPointer(
             new Mixxx::SoundSourceProviderCoreAudio));
 #endif
-#ifdef __FFMPEGFILE__
-    // FFmpeg currently overrides all other built-in providers
-    // if enabled
-    s_soundSourceProviders.registerProvider(Mixxx::SoundSourceProviderPointer(
-            new Mixxx::SoundSourceProviderFFmpeg));
-#endif
 
     // Scan for and initialize all plugins.
     // Loaded plugins will replace any built-in providers
     // that have been registered before (see above)!
     const QList<QDir> pluginDirs(getSoundSourcePluginDirectories());
-    foreach (QDir dir, pluginDirs) {
-        qDebug() << "Loading SoundSource plugins" << dir.path();
-        const QStringList files(dir.entryList(
+    for (const auto& pluginDir: pluginDirs) {
```

Comment:
Nit: Google C++ style uses a space before the ":"
I like that, because it gives a greater visual difference to ";" which has no space. 
There is no need, to change this in the PR. I just want to check if this can be a common sense for new code.